src/niftyone/analysis_levels/participant.py

```diff
@@ -5,20 +5,38 @@
 from concurrent.futures import ProcessPoolExecutor, as_completed
 from functools import partial
 from pathlib import Path
+from typing import Any
 
 import numpy as np
+import pkg_resources  # type:ignore [import-untyped]
```

Comment:
I think they removed this one front Setuptools (or deprecated it ?) importlib.metadata / importlib.resource I think are the new apis

Reply:
This is fixed in 7aaa470. Had to add a `type: ignore` here for mypy with different expected types than what resources returned.